docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -73,6 +73,36 @@ predictions are combined.
 you must restart the {dfanalytics-job}.
 
 
+[[dfa-regression-feature-encoding]]
+=== Feature encoding
```

Comment:
This looks like the same information as above.  If so, I think it should be re-used instead of restated in both places.  You might also consider moving this to a separate concept (like feature importance).  The regression and classification concepts feel like they're getting overlarge.